docs/user-guide/cli-authentication-methods.md

```diff
@@ -0,0 +1,288 @@
+
+1. Note the complete path for a service registered to API ML.
+
+    For example:
+
+    ```
+    https://myapilayerhost:port/<yourServiceId>/api/v1
+
+    ```
+
+    The format of base paths can vary based on how API ML is configured at your site.
+
+2. Using the example in Step 1, access the API ML instance by creating or updating a [service profile](../user-guide/cli-using-using-team-profiles.md#zowe-cli-profile-types), or issuing a command, with the base path value of `<yourServiceId>/api/v1`. Your service profile uses the token and credentials stored in your default base profile.
+
+    To create or update a service profile with the preceding base path in a **project** team configuration file:
+
+    ```
+    zowe config set profiles.<yourService>.properties.basePath <basePath> 
+    ```
+
+    If you are using a **global** team configuration file (located in your home directory), add `--global-config` to the end of the command.
+
+    Commands issued with this profile are routed through API ML to access an appropriate instance of your service.
+
+#### Specifying a base path with Zowe V1 profiles
+
+See the [Integrating with API Mediation Layer](https://67c89aa5af702da5881fc564--zowe-docs-master.netlify.app/v1.28.x/user-guide/cli-usingcli#integrating-with-api-mediation-layer) Zowe V1 documentation.
+
+## Using client certificates
+
+Certificates are a long lasting type of authentication, rather than a password or token that can expire in hours, days, or months. A certificate is authenticated by matching a public and private key.
+
+To use a client certificate for authentication:
+
+1. Configure the relevant profile with the paths to a public certificate file and private key file:
+
+   ```
+   zowe config set profiles.base.properties.certFile <certPath> 
+   ```
+
+   - `<certPath>`
+   
+      Specifies the location on your computer where the certificate is stored.
+
+2. Generate a file with the private key:
+   ```
+   zowe config set profiles.base.properties.certKeyFile <certKeyPath> 
+   ```
+      - `<certKeyPath>`
+   
+      Specifies the location on your computer where the private key is stored.
+
+
+:::tip
+Add the `--secure` option to the preceding commands to avoid saving certificate file paths as plain text in your configuration file.
+:::
+
+## Using multi-factor authentication (MFA)
+
+MFA adds an extra layer of security because it requires users to supply their password and an additional verification method.
+
+In Zowe, MFA can apply to basic authentication and single sign-on set with username and password.
+
+It is important to understand how MFA functions at your site in order to set it up. Typically, a user needs an authenticator app to obtain a temporary code that is typically concatenated or appended to the password. The system administrator must configure the application so it is set up for a Zowe user.
```

Comment:
"concatenated or appended to" seems redundant, should we say something like this?
```suggestion
It is important to understand how MFA functions at your site in order to set it up. Typically, a user needs an authenticator app to obtain a temporary code that is typically appended to the password or may replace it. The system administrator must configure the application so it is set up for a Zowe user.
```